src/getting-started.md

```diff
@@ -84,10 +84,20 @@ server=1
 rpcuser=username
 rpcpassword=password
 ```
-Run the Template provider:
+
+Unpack the Template Provider. For example, assuming you downloaded `bitcoin-sv2-tp-0.1.12-x86_64-linux-gnu.tar.gz`:
+
+```bash
+tar xvf bitcoin-sv2-tp-0.1.12-x86_64-linux-gnu.tar.gz
+cd sv2-tp-0.1.12
+```
+
+⚠️ Note: macOS binaries are not code signed. Read release notes for instructions on how to proceed.
+
+Start the Template Provider.
 
 ```bash
-./bitcoin-sv2-tp-0.1.*/bin/bitcoind -testnet4 -sv2 -sv2port=8442 -debug=sv2 
+./bitcoind -testnet4 -sv2 -sv2port=8442 -debug=sv2 
```

Comment:
I can only confirm this works, above, I can't test on MacOS. Thanks for pushing a PR.